src/main/java/com/ksm/wstbackoffice/controller/CountryController.java

```diff
@@ -37,10 +37,6 @@ public CountryController(CountryService countryService) {
 
         CountryDto country = countryService.findById(id);
```

Comment:
You don't need country variable if you don't use it. Just assign it directly in the response

Reply:
done